build/fpm/src/deb/config.ls.sh

```diff
@@ -13,6 +13,8 @@ pkg_config_dest=(
 
 pkg_depends=(
     "libc6 >= 2.17"
+    "libhiredis1.1.0 >= 1.1.0"
+    "libck0 >= 0.7.0"
```

Comment:
I don't think that `libhiredis1.1.0` is correct. It should be `libhiredis` right? Can you check what other packages define?